### PR TITLE
More updates and improvements

### PR DESCRIPTION
**Pull request by OgelGames**
A bunch of stuff, mostly tweaks and code cleanup, with some additions and improvements.

- Added setting to enable compression of picked-up item metadata, enabled by default.
- Added setting to change the number of times the wrench can be used.
- Created new textures, and added recipes for adding a colored grip to the wrench (see image below).
- Updated translations based on the commits in `technic`.
- Updated and improved the README and API documentation.
- Removed debug because it wasn't reliable (see todo below).

Example colored grip recipe:

image

Note the colored grip recipes make use of https://github.com/minetest/minetest/pull/12614 to change the item image of the wrench, so the recipes won't make much sense for older clients, but this will improve over time.

**TODO:**
- [x] Check if currently supported nodes' registrations are complete, as they might be missing metadata values if they were created using the debug. All possible node metadata values must be accounted for, otherwise pickup will be blocked.

### CODE REVIEW

**Review by BuckarooBanzay — APPROVED** [review]
didn't test but LGTM :+1:

**Comment by SwissalpS** [comment]
should I comment on translations here or do that in a subsequent PR?

**Comment by OgelGames** [comment]
> should I comment on translations here or do that in a subsequent PR?

Since most of the translations are empty, and it would be good to add them, I think that it would be better to do it all in another PR :)

File: README.md [hunk 1]
```diff
@@ -4,12 +4,16 @@
 [![License](https://img.shields.io/badge/license-LGPLv2.0%2B-purple.svg)](https://www.gnu.org/licenses/old-licenses/lgpl-2.0.en.html)
 [![ContentDB](https://content.minetest.net/packages/mt-mods/wrench/shields/downloads/)](https://content.minetest.net/packages/mt-mods/wrench/)
 
-The wrench tool allows you pickup nodes with inventories or other metadata, such as chests, signs, drawers, machines, etc, and transport them intact to another location.
+The wrench tool allows you pick up nodes with inventories or other metadata, such as chests, signs, drawers, machines, etc, and transport them intact to another location.
```
Comment (SwissalpS) [inline README.md]: ... allows you _**to**_ pick up nodes ...

It's correct in the mod.conf.
Personally I'd also drop the "you" or replace it with "player" but that isn't important, the missing "to" is.

Comment (OgelGames) [reply README.md]: Nice catch :)

**Comment by OgelGames** [comment]
I think everything for this PR is done, merging :)

**Pull request merged**